finetune/saver.py

```diff
@@ -136,11 +136,14 @@ def init_fn(scaffold, session):
             self.var_val = []
             for var in all_vars:
                 for saved_var_name, saved_var in itertools.chain(variables_sv.items(), self.fallback.items()):
-                    if saved_var_name == var.name:
+                    if saved_var_name == var.name or saved_var_name == var.name[:-2]: # change this back and fix bert base :
```

Comment:
Would appreciate a little bit of context here

Reply:
This needs removing. the first set of jl files I generated without `:0` on the end